lib/github/middleware.js

```diff
@@ -6,6 +6,12 @@ const { Subscription } = require('../models')
 const getJiraClient = require('../jira/client')
 const getJiraUtil = require('../jira/util')
 const enhanceOctokit = require('../config/enhance-octokit')
+
+let newrelic
+if (process.env.NEWRELIC_KEY) {
```

Comment:
I know you're just copying the pattern, but I'm wondering if we should use the [agent-enabled](https://docs.newrelic.com/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration#agent-enabled) configuration in [newrelic.js](https://github.com/integrations/jira/blob/master/newrelic.js#L8) and have it be `agent_enabled: !!process.env.NEWRELIC_KEY,` this would avoid this check and let use call out a noop to the newrelic agent (also we wouldn't have to do the if/else block below). What do you think? The other place I know this pattern is used is in [run.js](https://github.com/integrations/jira/blob/master/lib/run.js#L6).

We did this in [slack's newrelic.js](https://github.com/github/slack/blob/master/newrelic.js#L17-L18) and it's worked really well.

Reply:
Yeah I really didn't like this whole conditional stuff. That looks like a great way to simplify this code! I'll take a stab at it.

Reply:
@tebriel I think I'm going to punt this to a followup. Since it will just change how we load/conditional-ize this stuff. But the logic should still run.